src/providers/twitch/TwitchChannel.hpp

```diff
@@ -164,13 +173,16 @@ class TwitchChannel : public Channel, public ChannelChatters
     const QString &getDisplayName() const override;
     const QString &getLocalizedName() const override;
 
+    QString prepareMessage(Application *app, const QString &message) const;
```

Comment:
Yeah, I had added the argument because all two callsites already had a pointer to the application instance and it felt weird calling `getApp` again inside `prepareMessage`, but I agree it makes more sense to do so given it's a singleton.